src/TourManager/TourManager.Service.Implementation/TourService.cs

```diff
@@ -8,38 +9,13 @@
 
 namespace TourManager.Service.Implementation
 {
-    /// <summary>

```

Comment:
Please do not delete each other code, you can just hide comments if it not ok for you

Reply:
ITourService already contains all these comments, do we need the same for TourService?
I will return the comments anyway ;)